org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/XMLTextDocumentService.java

```diff
@@ -174,6 +174,11 @@ public void updateClientCapabilities(ClientCapabilities capabilities,
 			typeDefinitionLinkSupport = textDocumentClientCapabilities.getTypeDefinition() != null
 					&& textDocumentClientCapabilities.getTypeDefinition().getLinkSupport() != null
 					&& textDocumentClientCapabilities.getTypeDefinition().getLinkSupport();
+			sharedSettings.setOpenSettingsCommandSupport(
+					capabilities.getWorkspace() != null
+					&& capabilities.getWorkspace().getWorkspaceEdit() != null
+					&& capabilities.getWorkspace().getWorkspaceEdit().getResourceOperations() != null
+					&& capabilities.getWorkspace().getWorkspaceEdit().getResourceOperations().contains("create"));

```

Comment:
For future reference, we could use the predefined string `org.eclipse.lsp4j.ResourceOperationKind#Create` instead of `"create"`